json-schemas/src/app-stats.json

```diff
@@ -9,632 +9,786 @@
+      "description": "Unit of time for these stats from the intervalId forwards.",
       "pattern": "^minute$"
     },
     "schema": {
       "type": "string",
-      "description": "URI to this schema"
+      "description": "URI to this schema."
     },
     "entries": {
       "type": "object",
-      "description": "stats entries",
+      "description": "Stats entries.",
       "properties": {
         "messages.all.all.count": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total message count."
         },
         "messages.all.all.data": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total message size."
         },
         "messages.all.all.uncompressedData": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total uncompressed message size (excluding any compression, e.g., delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas)."
         },
         "messages.all.messages.count": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total message count, excluding presence messages."
         },
         "messages.all.messages.data": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total message size, excluding presence messages."
         },
         "messages.all.messages.uncompressedData": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total uncompressed message size (excluding any compression, e.g., delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas), excluding presence messages."
         },
         "messages.all.presence.count": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total presence message count."
         },
         "messages.all.presence.data": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total presence message size."
         },
         "messages.all.presence.uncompressedData": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total uncompressed presence message size (excluding any compression, e.g., delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas)."
         },
         "messages.inbound.realtime.all.count": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total inbound realtime message count (received by Ably from clients)."
```

Comment:
Here, and elsewhere, you refer to "Ably" when it would be clearer to refer to "the Ably service" - in order to make it certain that we're talking about "the server".